### ISSUE TITLE
No Midsummer or Christmas Eve

### ISSUE THREAD

**Issue by hugovk** [ericnishio/finnish-holidays#1]
```bash
$ finnish-holidays -y 2015
Thu, Jan  1, 2015 Uudenvuodenpäivä
Tue, Jan  6, 2015 Loppiainen
Fri, Apr  3, 2015 Pitkäperjantai
Mon, Apr  6, 2015 Toinen pääsiäispäivä
Fri, May  1, 2015 Vappu
Thu, May 14, 2015 Helatorstai
Fri, Dec 25, 2015 Joulupäivä

$ finnish-holidays -y 2015 -a
Thu, Jan  1, 2015 Uudenvuodenpäivä
Tue, Jan  6, 2015 Loppiainen
Sun, Feb  8, 2015 Kynttilänpäivä
Sun, Feb 15, 2015 Laskiaissunnuntai
Sun, Mar 22, 2015 Marian ilmestyspäivä
Sun, Mar 29, 2015 Palmusunnuntai
Fri, Apr  3, 2015 Pitkäperjantai
Sun, Apr  5, 2015 Pääsiäissunnuntai
Mon, Apr  6, 2015 Toinen pääsiäispäivä
Fri, May  1, 2015 Vappu
Thu, May 14, 2015 Helatorstai
Sun, May 17, 2015 Kaatuneiden muistopäivä
Sun, May 24, 2015 Helluntai
Sun, May 31, 2015 Pyhän Kolminaisuuden päivä
Sat, Jun 20, 2015 Juhannus
Sun, Jul  5, 2015 Apostolien päivä
Sun, Jul 19, 2015 Kirkastussunnuntai
Sun, Oct  4, 2015 Mikkelinpäivä
Sat, Oct 31, 2015 Pyhäinpäivä
Sun, Nov 22, 2015 Tuomiosunnuntai
Sun, Dec  6, 2015 Suomen itsenäisyyspäivä
Fri, Dec 25, 2015 Joulupäivä
Sat, Dec 26, 2015 Tapaninpäivä
```

They might not be official public holidays, but Midsummer Eve and Christmas Eve are important de facto holidays!

http://almanakka.helsinki.fi/en/flag-days-and-holidays-in-finland.html
https://en.wikipedia.org/wiki/Public_holidays_in_Finland

**Comment by ericnishio** [comment]
That's a great point, Hugo. Thanks! I'll add them to the list.

**Comment by ericnishio** [comment]
Fixed in 0.2.2.
